modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/data_sets.jsp

```diff
@@ -0,0 +1,47 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+portletDisplay.setBeta(true);
+%>
+
+<c:choose>
+	<c:when test='<%= FeatureFlagManagerUtil.isEnabled("LPD-15729") %>'>
+		<react:component
+			module="{DataSets} from frontend-data-set-views-web"
```

Comment:
There are too many differences in implementation for inline FF checks, in the end I chose separate implementations.